nixos/modules/services/home-automation/homebridge.nix

```diff
@@ -0,0 +1,549 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.homebridge;
+
+  args = concatStringsSep " " (
+    [
+      "-U ${cfg.userStoragePath}"
+      "-P ${cfg.pluginPath}"
+      "--strict-plugin-resolution"
+    ]
+    ++ optionals cfg.allowInsecure [ "-I" ]
+  );
```

Comment:
nit for readability: move this to where the `args` is used so it reads nicer

Reply:
Moved into the services' exec statements.